lib/config_upgrader.js

```diff
@@ -0,0 +1,272 @@
+"use strict";
```

Comment:
I'm a bit confused by this - I thought that ESLint had built-in functionality we could use for this. Why do we need this file?